### PR TITLE
feat: remove dynamic imports, import stigg components directly

### PR DESCRIPTION
**Pull request by antonzy**

### CODE REVIEW

**Comment by antonzy** [comment]
[Take a stab at fixing SSR when using React SDK](https://trello.com/c/sNBzKiah/2776-take-a-stab-at-fixing-ssr-when-using-react-sdk)

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@stigg/react-sdk": "0.16.0",
+    "@stigg/react-sdk": "latest",
```
Comment (itzik-bd) [inline package.json]: leftover?

File: pages/_app.tsx [hunk 2]
```diff
@@ -1,8 +1,24 @@
+import '@stigg/react-sdk/dist/styles.css';
 import '../styles/globals.css'
 import type { AppProps } from 'next/app'
+import {StiggProvider} from "@stigg/react-sdk";
+
+// Using the demo app API key, replace with your own client API key
+const STIGG_CLIENT_API_KEY = "$2b$10$ouA6M7FOJSxHH/3mkjZnZOJf5lUmFPvj8G0h1zIQxRkOY9CrpY53.:248ec79b-0932-4157-b514-30375bf5eeb8";
```
Comment (itzik-bd) [inline pages/_app.tsx]: I know it's not related, but worth to mention that prefer to read it from from `.env`

**Review by itzik-bd — COMMENTED** [review]
~Very nice 🦾

**Review by itzik-bd — COMMENTED** [review]
~Very nice 🦾

**Pull request merged**